### PR TITLE
[12.0][ADD] account_invoice_report_payment_info: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT24270

### CODE REVIEW

File: account_invoice_report_payment_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+=============
+
+* Activate developer mode.
+* Go to *Settings > Technical > Parameters > System Parameters*.
+* Locate the setting with key
+  "account_invoice_report_payment_info.info_pattern"
+  or create a new one if not exists.
+* Set a format pattern using the key available in _get_payments_vals method.
+  This module adds move_ref key to all those:
+  https://github.com/odoo/odoo/blob/ceb0c985afbbb41951c6f953648a6d0795d4e537/addons/account/models/account_invoice.py#L185
```
Comment (pedrobaeza) [inline account_invoice_report_payment_info/README.rst:46]: Please explain the available ones. This is a user readme, not technical one.

File: account_invoice_report_payment_info/README.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to **Invoicing > Customer Invoices**.
+#. Select or create an invoice.
+#. Create and validate credit note.
```
Comment (pedrobaeza) [inline account_invoice_report_payment_info/README.rst:55]: and the reconciliation?

**Review by sergio-teruel — APPROVED** [review]
Only a little question... Can you increase the test coverage??

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Thanks :+1:

**Pull request merged**